site/content/en/blog/releases/1.32.0.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "1.32.0 -  Enhanced Game Server Allocation, Go Version Upgrade and Stable Features!"
+linktitle: "1.32.0"
+date: "2023-05-23"
+---
+
+This is the 1.32.0 release of Agones.
+
+In this release, we are pleased to announce an upgrade to Go version 1.20. This upgrade ensures that your game servers are equipped with the latest tools and improvements, enhancing their overall performance and stability. We are also promoting some features up the feature stages, which means that they are now more stable and ready for use.
```

Comment:
I think it's worth specifically calling out the features that have graduated. For an example, see https://agones.dev/site/blog/2023/01/17/1.29.0-kubernetes-1.24.0-sdkgracefultermination-feature-in-beta-new-podhostname-feature/

Reply:
Does this description look good?

In this release, we are pleased to announce an upgrade to Go version 1.20. This upgrade ensures that your game servers are equipped with the latest tools and improvements, enhancing their overall performance and stability. We are also promoting ResetMetricsOnDelete, PodHostname, SplitControllerAndExtensions, SafeToEvict, and SDKGracefulTermination features up the feature stages, which means that they are now more stable and ready for use.

We encourage you to take advantage of these promoted features to enhance the stability, performance, and functionality of your game servers.

Reply:
SDKGracefulTermination is still in Beta as before, right? So it's not promoted up.

Reply:
Yes that's correct.. I have added the below description that does not include SDKGracefulTermination feature..

This is the 1.32.0 release of Agones.

In this release, we are pleased to announce an upgrade to Go version 1.20. This upgrade ensures that your game servers are equipped with the latest tools and improvements, enhancing their overall performance and stability. We are also promoting ResetMetricsOnDelete, PodHostname, SplitControllerAndExtensions, and SafeToEvict features up the feature stages, which means that they are now more stable and ready for use.

We encourage you to take advantage of these promoted features to enhance the stability, performance, and functionality of your game servers.

Reply:
This upgrade ensures that your game servers are equipped with the latest tools and improvements, enhancing their overall performance and stability:

* Upgrade to Go version 1.20. 
* The `ResetMetricsOnDelete` feature is now beta and enabled by default. Now when a Fleet or FleetAutoscaler is deleted from the system, Agones will automatically clear metrics that utilize their name as a label from the exported metrics, so the metrics exported do not continuously grow in size over the lifecycle of the Agones installation.
* The `PodHostname` feature that allows you to access a GameServer Pod within a cluster by a DNS name is now beta and enabled by default.
* The `SplitControllerAndExtensions` feature which splits the Agones controller into multiple binaries for better reliability is now beta and enabled by default.
* The `SafeToEvict` and `SDKGracefulTermination` features have both graduated to stable.

We encourage you to take advantage of these promoted features to enhance the stability, performance, and functionality of your game servers.

Reply:
Just a reminder that `SDKGracefulTermination` is still in beta